src/emails/templates/monthlyActivityFree/MonthlyActivityFreeEmail.tsx

```diff
@@ -81,28 +78,30 @@ export const MonthlyActivityFreeEmail = (
     // Show a sum of resolved data breach & broker exposures if a scan has been run
     // Otherwise, only show resolved data breaches
     dataPointValue: hasRunFreeScan
-      ? props.dataSummary.dataBreachResolvedNum +
-        props.dataSummary.dataBrokerManuallyResolvedNum
+      ? props.dataSummary.fixedSanitizedDataPoints.reduce(
+          (total, dataPointSummary) => {
+            return total + Object.values(dataPointSummary)[0];
+          },
+          0,
+        )
       : props.dataSummary.dataBreachResolvedNum,
```

Comment:
I'm not actually sure if `dataBreachResolvedNum` is the correct number to be looking at - looking at the `getDashboardSummary`, it's determined based on `breach.isResolved`, but as far as I know we're not really setting that anymore? Because we're not resolving breaches, but rather breached data types (except for passwords and security questions, which we do resolve per breach?).

Reply:
Thanks again Florian, and sorry for stirring up a panic @codemist!